README.md

```diff
@@ -454,6 +454,9 @@ This setting would be able to be overridden via the typical interest group
 mechanisms. For example, the update mechanism could support a new
 `privateAggregationConfig` key matching the call to `joinAdInterestGroup()`.
 
+See [allowlist](https://github.com/WICG/attribution-reporting-api/blob/main/aggregation_coordinator_origin_allowlist.md)
```

Comment:
i think it might make more sense to move this up just after line 415. We could also consider rewording to something like: "This allowlist matches the Attribution Reporting API's, available [here](URL)." Wdyt?

Reply:
That makes better sense - I made a new commit with the update, PTAL and see if that looks fine? 

Some of the language in the section around line 415 is "we propose", "we should", etc. If we've gone ahead with all these, should we update the language?

Reply:
Thanks, LGTM with one more nit.

Agreed that the language is a bit odd -- the explainer is meant for the "early/rough proposal" stage and isn't necessarily meant to be kept up-to-date like we've been doing. I think it's ok to leave as is given that, at least for now -- we're also trying to move our documentation to a more appropriate place.